packages/deno/src/generators/application/schema.json

```diff
@@ -40,9 +40,14 @@
     },
     "withWatch": {
       "type": "boolean",
-      "description": "Default the project to watch for file changes for the 'serve' target. For projects that do not need watch capability (eg: CLI projects), use --with-watch=false",
+      "description": "Default the project to watch for file changes for the 'serve' target. For projects that do not need watch capability (eg: CLI projects), use --with-watch=false.",
       "default": true
     },
+    "withDeploy": {
+      "type": "boolean",
```

Comment:
maybe an alias for --deno-deploy since that's the name of the service?

Reply:
Let's just rename it.